.travis.yml

```diff
@@ -26,6 +26,10 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
     - php: 5.6
+      env: SYMFONY_VERSION=3.1.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.2.*
+    - php: 5.6
```

Comment:
The build fail with `dev-master`.
I guess it's because it might be harder to resolve deps when it's the master branch.